README.md

```diff
@@ -38,3 +64,23 @@ zbf.load([workspaceId, formId, recordId]).then(function (data) {
 // You can also pass a long string if that's your thing.
 zbf.load('foo/bar/baz/' + workspaceId + '/etc');
 ```
+
+_Note about Deletions_
+
+Passing `null` as the second argument of `.save` will cause strange and unhelpful behavior.
+Instead, pass an object with the endpoint(s) as the key(s) and `null` as the value(s) for the second argument.
+See [Firebase docs](https://www.firebase.com/docs/web/api/firebase/update.html) for the underlying `.update()` command that is called.
+
+Ex:
+```js
+// much regrets
+zbf.save([workspaceId, 'undesiredField'], null).then(function () {
+    //etc...
+  })
+
+// all the good vibes
+zbf.save([workspaceId], {'undesiredField': null})
```

Comment:
@tehpsalmist just a minor note, if you only have 1 param you don't need to add it to an array you can just pass it directly to save()/load()

Reply:
Fixed. ;)